src/EntryList.js

```diff
@@ -0,0 +1,12 @@
+import App from "./App";
+function EntryList(props) {
+    return
+    (
+        <ul>
+            {fruits.map(function (fruit) {
```

Comment:
this is correct, but where `fruits` is defined?
there's something missing before `fruits`